README.md

```diff
@@ -15,7 +15,16 @@ Turn Twitch chatter into Python events.
 
 ## Usage
 
-### 1. Create an Observer
+### 1. Installation
+
+To install twitch-observer you need to download this GitHub repo and call `pip` in your shell of choice:
```

Comment:
Maybe you don't have to be so explicit here? Our target audience is developers. Perhaps something like:
> Clone this repo and run:
> ```pip install .```

This is only a suggestion, I'd like to hear your thoughts.